src/script/sign.cpp

```diff
@@ -33,14 +33,60 @@ bool MutableTransactionSignatureCreator::CreateSig(const SigningProvider& provid
     return true;
 }
 
+static bool GetCScript(const SigningProvider* provider, const SignatureData* sigdata, const CScriptID &scriptid, CScript& script)
```

Comment:
Commit "Replace CombineSignatures with ProduceSignature"

Looks like these could be refefenreces:
```cpp
static bool GetCScript(const SigningProvider& provider, const SignatureData& sigdata, const CScriptID &scriptid, CScript& script)
```
and remove `provider != nullptr` below.

Reply:
Otherwise fix space before `scriptid` argument. Same below in the definition.

Reply:
Done